Automattic/WP-Job-Manager
# Update and standardize asset and package build steps

### Changes proposed in this Pull Request

* !!! [Needs Testing] Switch from LESS to SCSS for base CSS syntax.
* Moves 3rd party library files to `assets/lib`.
* Removes all pre-built files.
* Removes `Gruntfile.js` and pretty much all build dependencies.
* Replaces with standardized scripts in `package.json` built around `@wordpress/scripts` (using webpack for the build) and `composer archive` for building the released archive.

### Testing instructions

I think we should merge this early in 1.35's cycle so we can test for visul regressions. I've done some testing so far and things seem fine.

* Run `composer install` and `npm install`.
* Run `npm run build` and use the `wp-job-manager.zip` to test the package, especially ensuring no CSS/visual regressions.
